src/Tracking/README.md

```diff
@@ -129,6 +129,16 @@ All event names are prefixed by `wcadmin_gla_`.
 
 -   `mc_account_reclaim_url_button_click` - Clicking on the button to reclaim URL for a Google Merchant Center account.
 
+-   `mc_phone_number_check` - Whether the phone number for Merchant Center exists or not.
```

Comment:
I find that description confusing. That makes me doubt the reason and timing of this event.

I think we'd need to describe when/why it's actually fired. This is just more of a description of events property.

1. If it's "Whenever we check the MC Account's phone number" - I'd move the `recordEvent` directly to `useGoogleMCPhoneNumber`. Then change the description to 
	  ```
	  Whenever the MC Account's phone number is checked.
	  ```
	  And changed `view` to `path`
2. If it's "Whenever phone number card is shown" I'd change the name & description to something like
	  ```
	  -   `mc_phone_number_information` - Whether the Phone Number Card is shown.
	  ```
	and move the trigger to `ContactInformation` `PhoneNumberCard`
3. If it's "Whenever contact information card is shown" I'd change the name & description to something like
	  ```
	  -   `mc_phone_number_information` - Whether the contact information card is shown.
	  ```

Reply:
@tomalec , it's actually more for "when we check the MC Account's phone number". However, `useGoogleMCPhoneNumber` is used in a few places and we don't want to track in all the places. So I think it is better not to put it inside `useGoogleMCPhoneNumber`.

The "Whenever phone number card is shown" and "Whenever contact information card is shown" isn't really that accurate, those sounds more like UI things which we are not really interested in.

> And changed `view` to `path`

That's a good one. I made an edit for that, in [`e27bb51` (#945)](https://github.com/woocommerce/google-listings-and-ads/pull/945/commits/e27bb51e0629581c5ca0eebf8227d7a8ce5865fb).

Reply:
> I made an edit for that, in e27bb51

:ok_hand: 

> The "Whenever phone number card is shown" and "Whenever contact information card is shown" isn't really that accurate, those sounds more like UI things which we are not really interested in.

But currently "Whenever contact information card is shown" is precisely when it happens now. And I think we should precisely document when events are being recorded so they could be used correctly.

Reply:
> it's actually more for "when we check the MC Account's phone number". However, `useGoogleMCPhoneNumber` is used in a few places and we don't want to track in all the places. So I think it is better not to put it inside `useGoogleMCPhoneNumber`.

I also think such an event brings the most value and flexibility for reporting.

From what I checked, currently, we use `useGoogleMCPhoneNumber` only in:
1. Contact information card - where we currently record
1. Contact information preview card - where I believe we could record
2. Step 4 of onboarding - where we indirectly record anyway (and with `path` change this trigger would be de-duped). Plus I believe that use is not needed ,and could be cleared up after the release.
![image](https://user-images.githubusercontent.com/17435/128749721-f13c82bf-ceff-4873-a1cb-5cfc1884336f.png)